packages/core/src/components/steps-modal/steps-modal.scss

```diff
@@ -0,0 +1,9 @@
+@import '~@atomium/scss-utils/index';
+
+.hide {
+  display: none;
+}
```

Comment:
(suggestion) I think more declarative control the `display: none` directly in the component, without any class. Using utility class just for it is not a good approach, normally it has a global scope

Reply:
done at [693539c](https://github.com/juntossomosmais/atomium/pull/599/commits/693539c1609736fc4c7eaffbc565d0631bc03425)